core/src/main/java/org/mule/runtime/core/internal/el/function/LookupFunction.java

```diff
@@ -82,11 +83,13 @@ public Object call(Object[] parameters, BindingContext context) {
     if (component instanceof Flow) {
       Message incomingMessage = lookupValue(context, MESSAGE, Message.builder().nullValue().build());
       Map<String, ?> incomingVariables = lookupValue(context, VARS, EMPTY_MAP);
+      Map<String, ?> incomingParams = lookupValue(context, PARAMS, EMPTY_MAP);
       Error incomingError = lookupValue(context, ERROR, null);
 
       Message message = Message.builder(incomingMessage).value(payload).mediaType(APPLICATION_JAVA).build();
       CoreEvent event = CoreEvent.builder(PrivilegedEvent.getCurrentEvent().getContext())
           .variables(incomingVariables)
+          .parameters(incomingParams)
```

Comment:
this is similiar to a flow-ref. Are we sure we want to forward the params as well? This could generate coupling on the target flow to its caller.

Reply:
you're right. We should actually add a validator to disalllow flow-ref inside operations